utility/src/main/java/org/oppia/android/util/parser/image/UrlImageParser.kt

```diff
@@ -160,7 +160,10 @@ class UrlImageParser private constructor(
               htmlContentTextView.paddingRight,
               htmlContentTextView.paddingBottom
             )
-          proxyDrawable.initialize(drawable, computeBounds(context, drawable, viewWidth, padding))
+          proxyDrawable.initialize(
+            drawable,
+            computeBounds(context, drawable, viewWidth, padding)
+          )
```

Comment:
Can the old code formatting be kept now?

Reply:
Done